src/vs/editor/contrib/find/browser/findController.ts

```diff
@@ -58,6 +60,47 @@ export function getSelectionSearchString(editor: ICodeEditor, seedSearchStringFr
 	return null;
 }
 
+export function getSelectionTextFromEditor(allowUnselectedWord: boolean, editor: IEditor, configurationService: IConfigurationService): string | null {
```

Comment:
Can you put this function is `SearchView.ts`? It's only used by search code, so it should be near the related code.